sdk/trace/src/test/java/io/opentelemetry/sdk/trace/data/ImmutableStatusDataTest.java

```diff
@@ -23,15 +23,4 @@ void defaultConstants() {
     assertThat(StatusData.error().getStatusCode()).isEqualTo(StatusCode.ERROR);
     assertThat(StatusData.error().getDescription()).isEmpty();
   }
-
-  @Test
-  void generateCodeToStatus() {
```

Comment:
We don't need this since errorprone will make sure the compile fails if a new enum value is added.

Reply:
Frankly I'd prefer to rely on our own tests rather than on a build tool configuration. But perhaps I'm too defensive :)

Reply:
alternatively, we could *only* have this test method, which covers everything in the method above and wouldn't require adding more entries if we add another status.

Reply:
Sure combined into one test